fe/fe-core/src/main/java/com/starrocks/catalog/ResourceGroup.java

```diff
@@ -56,9 +57,19 @@ public class ResourceGroup {
     static {
         DEFAULT_WG.setId(DEFAULT_WG_ID);
         DEFAULT_WG.setName(DEFAULT_RESOURCE_GROUP_NAME);
+        DEFAULT_WG.setResourceGroupType(TWorkGroupType.WG_DEFAULT);
+        DEFAULT_WG.setConcurrencyLimit(0);
+        DEFAULT_WG.setMemLimit(1.0);
+        DEFAULT_WG.setCpuCoreLimit(BackendCoreStat.getAvgNumOfHardwareCoresOfBe());
+        DEFAULT_WG.setMaxCpuCores(BackendCoreStat.getAvgNumOfHardwareCoresOfBe());
 
         DEFAULT_MV_WG.setId(DEFAULT_MV_WG_ID);
         DEFAULT_MV_WG.setName(DEFAULT_MV_RESOURCE_GROUP_NAME);
+        DEFAULT_MV_WG.setResourceGroupType(TWorkGroupType.WG_MV);
+        DEFAULT_MV_WG.setConcurrencyLimit(0);
+        DEFAULT_MV_WG.setCpuCoreLimit(BackendCoreStat.getAvgNumOfHardwareCoresOfBe());
+        DEFAULT_MV_WG.setMaxCpuCores(BackendCoreStat.getAvgNumOfHardwareCoresOfBe());
+        DEFAULT_MV_WG.setMemLimit(0.8);
```

Comment:
it should not be hardcoded

Reply:
Yes, `BE` should be queried to get the config value.